### PR TITLE
Minor Changes:

### PR DESCRIPTION
**Pull request by arturobernalg**
* Simplify conditions.
* Remove unnecessary 'null' check.

### CODE REVIEW

File: src/main/java/org/apache/commons/ognl/ASTAdd.java [hunk 1]
```diff
@@ -129,13 +129,6 @@ boolean isWider( NodeType type, NodeType lastType )
         {
             return false;
         }
-
-        if ( Float.class.isAssignableFrom( type.getGetterClass() )
-            && Integer.class.isAssignableFrom( lastType.getGetterClass() ) )
-        {
-            return true;
-        }
```
Comment (lukaszlenart) [inline src/main/java/org/apache/commons/ognl/ASTAdd.java:137]: Why did you remove this?

File: src/main/java/org/apache/commons/ognl/ASTVarRef.java [hunk 2]
```diff
@@ -124,7 +124,7 @@ public String toGetSourceString( OgnlContext context, Object target )
             post = ")";
         }
 
-        if ( parent != null && ASTAssign.class.isInstance( parent ) )
+        if (ASTAssign.class.isInstance(parent))
```
Comment (lukaszlenart) [inline src/main/java/org/apache/commons/ognl/ASTVarRef.java:127]: Why not to use `isinstanceof`?

File: src/main/java/org/apache/commons/ognl/ASTAdd.java (hunk 1 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/ognl/ASTAdd.java:137]: > Why did you remove this?

Hi. @lukaszlenart 
If the code get there, always will return true


 ```
 if ( Float.class.isAssignableFrom( type.getGetterClass() )
            && Integer.class.isAssignableFrom( lastType.getGetterClass() ) )
        {
            return true;
        }

        return true;
```

**Review by arturobernalg — COMMENTED** [review]
HI @lukaszlenart 
I was waiting for the previous PR. Now we no need this PR 
TY

File: src/main/java/org/apache/commons/ognl/ASTAdd.java (hunk 1 above)
Comment (lukaszlenart) [reply src/main/java/org/apache/commons/ognl/ASTAdd.java:137]: Right, but I think it can also be misleading why there is no `Float` and `Integer` check as above.

**Comment by lukaszlenart** [comment]
I think you need to rebase this PR over master branch
